app/models/order.rb

```diff
@@ -1,4 +1,10 @@
 class Order < ApplicationRecord
   has_many :cart_items, dependent: :destroy
   belongs_to :user, optional: true
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z]+)*\.[a-z]+\z/i
```

Comment:
https://github.com/ergoserv/handbook/blob/master/guides/models.md#template check for correct order